src/components/GameBoard/GameBoard.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { createRef, useEffect } from 'react';
+import './GameBoard.css';
```

Comment:
Лучше 

```suggestion
import css from './GameBoard.css';
```

и затем в коде использовать `css.game`, `css.correct` и т. д.